metadata.json

```diff
@@ -41,6 +41,16 @@
         "9"
       ]
     },
+    {
+      "operatingsystem": "Debian",
+      "operatingsystemrelease": [
+        "11",
+        "12"
```

Comment:
Puppet packages for Debian 12 are not yet available, so we cannot test this ATM in our CI. I would recommend to add support when we can test it (I think we can keep the code and only add the info in metadata.json when ready).

Reply:
Works for me on 12 in my laptop :-)

Seriously though t'other day was as asked to add F38 after initially only adding F36 because AIO puppet did not exist.

Given we don't dictate use of aio puppet over any other source of puppet it seems valid to me to claim support even with out tests.